Project/src/MakeCall/CallCard.js

```diff
@@ -871,11 +897,11 @@ export default class CallCard extends React.Component {
                             variant="secondary"
                             onClick={() => this.handleVideoOnOff()}>
                             {
-                                this.state.videoOn &&
+                                this.state.canOnVideo && this.state.videoOn &&
```

Comment:
We discussed this comment, the feature usage, is to reflect the right UI based on the current capabilities, rather than clicking on a button and erroring out user experience. The PR reflects this practical usage of capabilities features.